src/containers/App/App.js

```diff
@@ -22,11 +21,11 @@ export class App extends Component {
   render() {
     return (
       <div className="App">
-        <Switch>
+        <Switch>//Remove this and make login a popup!!
```

Comment:
That would be great, a better user experience overall since the login is visually such a small component anyway!